hydrolib/hydromt_delft3dfm/workflows/graphs.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+import configparser
+import logging
+
+import geopandas as gpd
+import hydromt.io
+import numpy as np
+import pandas as pd
+import shapely
+from hydromt import config
+import networkx as nx
+
+logger = logging.getLogger(__name__)
+
+
+def gpd_to_digraph(data: gpd.GeoDataFrame) -> nx.DiGraph():
+
+    _ = data.copy()
```

Comment:
would just call this var `copy` or `data_copy` for example, to make it a little more readable.